phenol-core/src/main/java/org/monarchinitiative/phenol/formats/hpo/HpoAnnotation.java

```diff
@@ -24,7 +26,7 @@
   /** The characteristic age of onset of a feature in a certain disease. */
   private final HpoOnset onset;
   /** List of modifiers of this annotation. List can be empty but cannot be null */
-  private final List<TermId> modifierList;
+  private final List<TermId> modifiers;
 
   /** If no information is available, then assume that the feature is always present! */
   private static final HpoFrequency DEFAULT_HPO_FREQUENCY = HpoFrequency.ALWAYS_PRESENT;
```

Comment:
These constants aren't required any more as they are encapsulated in the Builder